src/firebase.android.ts

```diff
@@ -1,4 +1,5 @@
 import { DocumentSnapshot, firebase, GeoPoint, QuerySnapshot } from "./firebase-common";
+import * as firebaseMessaging from "./messaging/messaging";
```

Comment:
I'm getting `firebase.android.ts:2:36 - error TS2307: Cannot find module './messaging/messaging'.`, I think there's an `index.d.ts` missing in Git. Probably because my rules in the `.gitignore` file are a bit silly (I'm ignoring those files in there without a wildcard).